extensions/wikia/WikiFactory/SpecialWikiFactory_body.php

```diff
@@ -326,152 +361,155 @@ public function doWikiForm() {
+					$vars[ 'lookupuser_by_founder_usermail_url' ] = Title::newFromText( "LookupUser", NS_SPECIAL )->getFullURL( [  "target" => $vars['founder_usermail']  ] );
 				}
 			}
 		}
-		if( $this->mTab === "tags" ||  $this->mTab === "findtags" ) {
+		if ( $this->mTab === "tags" ||  $this->mTab === "findtags" ) {
 			$vars[ 'searchTag' ] = $this->mSearchTag;
 			$vars[ 'searchTagWikiIds' ] = $this->mTagWikiIds;
 		}
-		if( $this->mTab === "hubs" ) {
+		if ( $this->mTab === "hubs" ) {
 
 			$hub = WikiFactoryHub::getInstance();
 			$vars['vertical_id'] = $hub->getVerticalId( $this->mWiki->city_id );
 			$vars['verticals'] = $hub->getAllVerticals();
 
 			$wiki_old_categories = $hub->getWikiCategories ( $this->mWiki->city_id, false );
 			$wiki_new_categories = $hub->getWikiCategories( $this->mWiki->city_id, true );
-			$wiki_categories = array_merge($wiki_old_categories, $wiki_new_categories);
+			$wiki_categories = array_merge( $wiki_old_categories, $wiki_new_categories );
 
-			$wiki_cat_ids = array();
-			foreach ($wiki_categories as $cat) {
+			$wiki_cat_ids = [];
+			foreach ( $wiki_categories as $cat ) {
 				$wiki_cat_ids[] = $cat['cat_id'];
 			}
 			$vars['wiki_categoryids'] = $wiki_cat_ids;
 
 			$all_old_categories = $hub->getAllCategories( false );
 			$all_new_categories = $hub->getAllCategories( true );
-			$all_categories = array_replace($all_old_categories, $all_new_categories);
+			$all_categories = array_replace( $all_old_categories, $all_new_categories );
 
 			$vars['all_categories'] = $all_categories;
 
 		}
-		if( $this->mTab === "clog" ) {
+		if ( $this->mTab === "clog" ) {
 			$pager = new ChangeLogPager( $this->mWiki->city_id );
-			$vars[ "changelog" ] = array(
+			$vars[ "changelog" ] = [
 				"limit"     => $pager->getForm(),
 				"body"      => $pager->getBody(),
 				"nav"       => $pager->getNavigationBar()
-			);
+			];
 		}
-		if( $this->mTab === "ezsharedupload" ) {
-			global $wgServer;
-			$vars[ "EZSharedUpload" ] = array(
-				"active" => WikiFactory::getVarValueByName( "wgUseSharedUploads", $this->mWiki->city_id ),
-				"varTitle" => Title::makeTitle( NS_SPECIAL, 'WikiFactory' )->getFullUrl() . ( "/" . $this->mWiki->city_id . "/variables/" ),
-				"info" => ( isset($info) ? $info : "" ),
-				"local"  => array(
+		if ( $this->mTab === "ezsharedupload" ) {
+			$vars[ "EZSharedUpload" ] = [
+				"active" => WikiFactory::getVarValueByName(
+					"wgUseSharedUploads",
+					$this->mWiki->city_id
+				),
+				"varTitle" => Title::makeTitle( NS_SPECIAL, 'WikiFactory' )->getFullURL() .
+				              "/" . $this->mWiki->city_id . "/variables/",
+				"info" => ( isset( $info ) ? $info : "" ),
+				"local"  => [
 					"wgServer" => $this->mWiki->city_url,
 					"wgSharedUploadDBname" => WikiFactory::getVarValueByName( "wgSharedUploadDBname", $this->mWiki->city_id ),
 					"wgSharedUploadDirectory" => WikiFactory::getVarValueByName( "wgSharedUploadDirectory", $this->mWiki->city_id ),
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "wgSharedUploadDirectory" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)